### PR TITLE
Update mendix-profile.md

### PR DESCRIPTION
**Pull request by hbvndrzwt**
Added a section to describe the create account from profile flow. Also updated the change password and forget account actions.

### CODE REVIEW

File: content/en/docs/developerportal/community-tools/mendix-profile.md [hunk 1]
```diff
@@ -291,15 +291,33 @@ The following details may have conflicts, but these can easily be updated after
...
 
-### 10.2 Account
+### 10.2 Accounts
 
 This page is an overview of all the accounts you own and that are linked to your Mendix Profile. You can see which account you are logged in with, the company associated with each user account, if single sign-on is enabled, and the status of each user account (**Active** or **Deactivated**).
 
 By default, you are requested to change your password every 90 days. A Mendix Admin can change this password policy setting in [Control Center](/control-center/#security).
 
-You can update the password of each of your accounts via the [Change Password](https://login.mendix.com/mxid3/request-password-reset) button, which you can also reach using the **Forgot password?** link on the login screen.
+You can update the password of your logged-in account via the [Change Password](https://login.mendix.com/mxid3/request-password-reset) button that is available by clicking the menu button of the logged-in account. You can also reach this screen using the **Forgot password?** link on the login screen.
```
Comment (Adam-Dupaski) [inline content/en/docs/developerportal/community-tools/mendix-profile.md]: @hbvndrzwt how do you find the Change Password button - via which "menu button of the logged-in account" ? I can't access it in my own profile.

Comment (Adam-Dupaski) [reply content/en/docs/developerportal/community-tools/mendix-profile.md]: Nevermind, same button as for "Delete This Account" - (**...**)

File: content/en/docs/developerportal/community-tools/mendix-profile.md [hunk 2]
```diff
@@ -291,15 +291,35 @@ The following details may have conflicts, but these can easily be updated after
...
 
 By default, you are requested to change your password every 90 days. A Mendix Admin can change this password policy setting in [Control Center](/control-center/#security).
 
-You can update the password of each of your accounts via the [Change Password](https://login.mendix.com/mxid3/request-password-reset) button, which you can also reach using the **Forgot password?** link on the login screen.
+You can update the password of your logged-in account via the [Change Password](https://login.mendix.com/mxid3/request-password-reset) button that is available by clicking the menu button (**...**) of the logged-in account. You can also reach this screen using the **Forgot password?** link on the login screen.
+
+If you do not want to use an account anymore, you can remove it from this page. Click the menu button (**...**) for the logged-in account and select **Forget this Account**. All of the account information is then removed and deleted from Mendix's servers. All the personal and identifiable information of your profile is deleted when you delete all of your accounts.
```
Comment (Adam-Dupaski) [inline content/en/docs/developerportal/community-tools/mendix-profile.md]: @hbvndrzwt this is still **Delete this account** in my profile - has this update with the changed button name been released yet?

Comment (hbvndrzwt) [reply content/en/docs/developerportal/community-tools/mendix-profile.md]: You are right, it should be **Delete this account** instead. **Forget this Account** was the old text. I will update it to Delete.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**